### PR TITLE
Adding validation of EML document with R

### PR DESCRIPTION
**Pull request by yvanlebras**
as it appears to me that reading the initial doc was making me thinking that you need to use java or command line. Moreover, fix a typo.

THANK YOU FOR this amaziniinininnng work on this new EML release + for the EML doc!!!!

### CODE REVIEW

File: docs/eml-validation-refs.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ Because some of these rules cannot be enforced in XML-Schema, we have
 written a parser which checks the validity of the references and `id`s
 used in a document. This parser is included with the release of
 EML. To run the parser, you must have Java installed. To execute
-it change into the top-lvel directory of the EML release and run the
-'validate.sh' script passing your EML instance file as a parameter.
+it change into the top-level directory of the EML release and run the
+'validate.sh' script passing your EML instance file as a parameter. You can also usinvalidate your EML document from R, using the EML::validate() function.
```
Comment (mbjones) [inline docs/eml-validation-refs.md]: @yvanlebras Thanks for the editing. Happy to merge this in, but I'm unclear what you mean by 'usinvalidate'. Did you mean 'You can also validate your EML...'?

Comment (yvanlebras) [reply docs/eml-validation-refs.md]: Yes, sorry... My sausages-like fingers for sure  ;)

Comment (yvanlebras) [reply docs/eml-validation-refs.md]: Is it ok for you?

Comment (mbjones) [reply docs/eml-validation-refs.md]: That works for me. Thanks.

**Comment by mbjones** [comment]
@yvanlebras Thanks for your contribution. Changes have been deployed to the EML site.

**Comment by yvanlebras** [comment]
Thank you Matt, tank you for your eagle eye ;) THis is a really little contribution, really thank you for your amazing work, to you and colleagues around eml, and related products/services, and for this new documentation website!!!!!

**Pull request merged**